risc0/zkp/src/verify/mod.rs

```diff
@@ -320,10 +320,10 @@ where
         // tracing::debug!("Result = {result:?}");
 
         // Now generate the check polynomial
-        // TODO: This currently treats the extension degree as hardcoded at 4, with
+        // TODO: This currently treats the expansion degree as hardcoded at 4, with
```

Comment:
"expansion degree" isn't a term i think we should be using; `degree` can refer to a feature of a polynomial or a feature of an extension field, but if we're talking about the reed-solomon expansion, `rate` or `blow-up factor` is more appropriate. IIUC, what we're referencing here is called `invRate` in the codebase (for inverse of rate)

Reply:
This was recommended by @jbruestle. Feel free to rewrite any of these comments.

BTW, this is a draft and it's not ready yet for review. I only pushed it so I could share what I have to get help getting inverse worked out.

Reply:
Agree it's probably 'expansion factor', but also, this is a very 'draft' PR at this point, so I wouldn't sweat it.